### PR TITLE
pass pre-commit tests

### LINKED ISSUE #274: pass pre-commit checks
> Since unit tests have been passing before this branch, comment out parts so that tests pass now as well.

In a separate issue, resolve issues [noted](https://github.com/bluesky/hklpy/issues/263#issuecomment-1766882681) when certain tests were commented out due to failure with existing code.

_Originally posted by @prjemian in https://github.com/bluesky/hklpy/issues/272#issuecomment-1766887521_

### PR DESCRIPTION
**Pull request by prjemian**
- close #274

### CODE REVIEW

**Comment by prjemian** [comment]
In draft waiting on #282, #284, and #285.

**Comment by prjemian** [comment]
Needs `apischema` in the environment files.  That's coming from PR #285.

File: hkl/__init__.py [hunk 1]
```diff
@@ -30,68 +32,59 @@
...
+from .geometries import SimulatedK6C
+from .geometries import SoleilMars
+from .geometries import SoleilNanoscopiumRobot
+from .geometries import SoleilSiriusKappa
+from .geometries import SoleilSiriusTurret
+from .geometries import SoleilSixsMed1p2
+from .geometries import SoleilSixsMed2p2
+from .geometries import SoleilSixsMed2p3
+from .geometries import SoleilSixsMed2p3v2
+from .geometries import Zaxis
```
Comment (mrakitin) [inline hkl/__init__.py:59]: Is it what `isort` recommends doing?

**Review by mrakitin — APPROVED** [review]
Looks great! Thanks, @prjemian!

File: hkl/__init__.py (hunk 1 above)
Comment (prjemian) [reply hkl/__init__.py:59]: It's an isort preference.    https://github.com/bluesky/hklpy/blob/706295113424e2af54d8922df173c73d31baf8fc/pyproject.toml#L104C1-L105C1

```ini
[tool.isort]
profile = "black"
force_single_line  = "True"
line_length = 88
multi_line_output = "NOQA"
src_paths = ["hkl"]
```

Comment (prjemian) [reply hkl/__init__.py:59]: The `# noqa` was not copied to the other lines.  But still passes.

**Pull request merged**